### DISCUSSION TITLE
Feature: Jokes view count

### DISCUSSION THREAD

**Discussion by oguzhanaksoy** [Code2Gether-Discord/JokesOnYou#146] [Ideas]
I shared a joke but most people don't click the vote button. 
They will look laugh (or don't) and they will just pass. 
So, keep the user happy and analytics as how many people I reach, we can put View Counter.

we need to make sure that we are not making forever view increment by refreshing the browser.

**Comment by chrisK00** [comment]
Are you thinking that the counter goes up once when the user views the punchline?
I think this is a cool idea for the future though, as long as it doesnt take much time since i dont consider it very usefull because we wont know if the joke was actually good just by knowing many people viewed it.
